pallets/portal/rpc/runtime-api/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+authors     = [ "t3rn ltd. <team@t3rn.io>" ]
+description = "t3rn portal RPC Runtime API"
+edition     = "2018"
+homepage    = "https://t3rn.io"
+license     = "Apache 2.0"
+name        = "pallet-portal-rpc-runtime-api"
+readme      = "../README.md"
+repository  = "https://github.com/t3rn/t3rn/"
+version     = "1.0.0-alpha.0"
+
+[package.metadata.docs.rs]
+targets = [ "x86_64-unknown-linux-gnu" ]
+
+[dependencies]
+codec                       = { package = "parity-scale-codec", version = "3", default-features = false, features = [ "derive" ] }
+pallet-portal                 = { path = "../..", default-features = false }
```

can easily format these with `taplo format .`
```suggestion
pallet-portal              = { path = "../..", default-features = false }
```

---

obviously there was an automated solution for this ^^